test/simple.js

```diff
@@ -73,41 +73,43 @@ test.cb('should not delete a fork that has more branches', (t) => {
 	});
 });
 
-// Currently our logic does not delete this fork
-test.skip('should delete a fork that has more branches, but all at upstream branch tips', (t) => {
-	const mock = lib.mock({
-		listBranches(arguments_) {
-			if (arguments_.user === lib.USER.login) {
+test.cb(
+	'should delete a fork that has more branches, but all at upstream branch tips',
```

Comment:
eslint strangely formats this. If the `test.cb` part is anything other than `test.skip`, it puts arguments on separate lines. If it’s `test.skip`, then both arguments are on one line. That’s with ava/no-skip-test disabled.

At first I thought it’s due to the line length, but no, even in the PR diff the shorter version is wrapped, and the longer isn’t. And, also, `test.ski` is wrapped, and `test.skipp` is wrapped. Weird.

Reply:
That’s Prettier. Yeah I don’t like Prettier’s wrapping either 😉 
XO doesn’t care about line length as long as there’s no more than one `;` per line.

Reply:
You’re right that it’s prettier! https://github.com/prettier/prettier/issues/9285